.github/workflows/test-sdk.yml

```
@@ -0,0 +1,61 @@
+name: Test SDK (Reusable)
+
+on:
```

YAMLlint (1.37.1)

[warning] truthy value should be one of [false, true]

(truthy)